contrib/podmanimage/upstream/Dockerfile

```diff
@@ -0,0 +1,66 @@
+# git/Dockerfile
+#
+# Build a Podman container image from the latest
+# upstream version of Podman on GitHub.
+# https://github.com/containers/libpod
+# This image can be used to create a secured container
+# that runs safely with privileges within the container.
+# The containers created by this image also come with a
+# Podman development environment in /root/podman.
+#
+FROM fedora:latest
+ENV GOPATH=/root/podman
+
+# Install the software required to build Podman.
+# Then create a directory and clone from the Podman
+# GitHub repository, make and install Podman
+# to the container.
+# Finally remove the podman directory and a few other packages
+# that are needed for building but not running Podman
+RUN dnf -y install --enablerepo=updates-testing \
+     atomic-registries \
+     btrfs-progs-devel \
+     conmon \
+     containernetworking-cni \
+     device-mapper-devel \
+     git \
+     glib2-devel \
+     glibc-devel \
+     glibc-static \
+     go \
+     golang-github-cpuguy83-go-md2man \
+     gpgme-devel \
+     iptables \
+     libassuan-devel \
+     libgpg-error-devel \
+     libseccomp-devel \
+     libselinux-devel \
+     make \
+     ostree-devel \
+     pkgconfig \
+     runc \
+     containers-common; \
+     mkdir /root/podman; \
+     git clone https://github.com/containers/libpod /root/podman/src/github.com/containers/libpod; \
+     cd /root/podman/src/github.com/containers/libpod; \
+     make;\
+     make install;\
+     rm -rf /root/podman/*; \
+     dnf -y remove bats git golang go-md2man make; \
+     dnf clean all;
+
+
+
+
+# Adjust storage.conf to enable Fuse storage.
+RUN sed -i -e 's|^#mount_program|mount_program|g' -e '/additionalimage.*/a "/var/lib/shared",' /etc/containers/storage.conf
+RUN mkdir -p /var/lib/shared/overlay-images /var/lib/shared/overlay-layers; touch /var/lib/shared/overlay-images/images.lock; touch /var/lib/shared/overlay-layers/layers.lock
+
+# This needs to be fixed in podman and container/storage, but for now
```

Comment:
@TomSweeneyRedHat Are you sure we still need this in buildah as well as podman?

Reply:
Not sure at all @rhatdan.  I wasn't aware that this wasn't necessary now, will try removing it.